project/Http4sPlugin.scala

```diff
@@ -67,39 +67,36 @@ object Http4sPlugin extends AutoPlugin {
 
       IO.write(dest, buildData)
     },
+
     // servlet-4.0 is not yet supported by jetty-9 or tomcat-9, so don't accidentally depend on its new features
     dependencyUpdatesFilter -= moduleFilter(organization = "javax.servlet", revision = "4.0.0"),
     dependencyUpdatesFilter -= moduleFilter(organization = "javax.servlet", revision = "4.0.1"),
     // servlet containers skipped until we figure out our Jakarta EE strategy
-    dependencyUpdatesFilter -= moduleFilter(
-      organization = "org.eclipse.jetty*",
-      revision = "10.0.*"),
-    dependencyUpdatesFilter -= moduleFilter(
-      organization = "org.eclipse.jetty*",
-      revision = "11.0.*"),
-    dependencyUpdatesFilter -= moduleFilter(
-      organization = "org.apache.tomcat",
-      revision = "10.0.*"),
+    dependencyUpdatesFilter -= moduleFilter(organization = "org.eclipse.jetty*", revision = "10.0.*"),
+    dependencyUpdatesFilter -= moduleFilter(organization = "org.eclipse.jetty*", revision = "11.0.*"),
+    dependencyUpdatesFilter -= moduleFilter(organization = "org.apache.tomcat", revision = "10.0.*"),
     // Cursed release. Calls ByteBuffer incompatibly with JDK8
     dependencyUpdatesFilter -= moduleFilter(name = "boopickle", revision = "1.3.2"),
+
     headerSources / excludeFilter := HiddenFileFilter ||
       new FileFilter {
-        def accept(file: File) =
+        def accept(file: File) = {
           attributedSources.contains(baseDirectory.value.toPath.relativize(file.toPath).toString)
+        }
 
         val attributedSources = Set(
```

Comment:
I wonder whether we even need this hack if sbt-header is leaving the attributions alone now?  That would be lovely, because you just experienced firsthand how fragile this is.

Reply:
We don't (assuming you are happy with the dual-headers), I can remove it.

Reply:
Yeah, what you did looks great.  People have tried to tear this project down for stealing work over the years, but I don't think many projects go to our lengths to credit their sources.  Thanks for persevering on this to keep that tradition alive.

Reply:
I had no idea about that history, really sad to hear because it's readily apparent you've gone above and beyond here.